src/ray/raylet/scheduling/cluster_task_manager.h

```diff
@@ -97,6 +97,12 @@ class ClusterTaskManager {
   /// false if the task is already running.
   bool CancelTask(const TaskID &task_id);
 
+  /// Populate the relevant parts of the heartbeat table. This is intended for
+  /// sending raylet <-> gcs heartbeats. In particular, this should fill in
+  /// resource_load and resource_load_by_shape.
+  void Heartbeat(std::shared_ptr<HeartbeatTableData> data,
```

Comment:
Can you document the args?

Also, I think the convention is to list output parameters after input parameters.